schema/xsd/direct.xsd

```diff
@@ -59,6 +59,7 @@
                 </xs:element>
             </xs:choice>
             <xs:element name="signature-type" type="signature-type" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="on-behalf-of" type="signing-on-behalf-of" minOccurs="0" maxOccurs="1"/>
```

Comment:
Gjort noen tanker om å bruke `default="SELF"`? Det er litt pros and cons såklart. Man får default-verdien til å formelt bli en del av APIet, og ved å bruke [JAXB default value plugin](https://java.net/projects/jaxb2-commons/pages/Default-Value) kan man følgelig alltid hente ut en verdi herfra, og koden trenger dermed ikke å ha noe kunnskap om hva som er default-verdi.

Det negative er kanskje at det er litt guffent å ev. endre default-verdien hvis man trenger det.